README.md

```diff
@@ -24,9 +24,30 @@ $ ialirt_data_access --url <url> ialirt-log-query --year <year> --doy <doy> --in
 Download a log and place in Downloads directory or optionally specify another local directory by appending --downloads_dir <directory> to the command
 
 ```bash
-$ ialirt_data_access --url <url> ialirt-log-download ----filename <filename>
+$ ialirt_data_access --url <url> ialirt-log-download --filename <filename>
 ```
 
+### Query the database
+
+Query the database for a given time and/or data product. Examples shown below.
+
+```bash
+$ ialirt_data_access --url <url> ialirt-db-query --insert_time_start <insert_time_start> --insert_time_end <insert_time_end> --product_name <product_name>
```

Comment:
We are using `imap-data-access` with hyphens on the other CLI tool. Can we change this to be the same? Edit: I think it already is, so these should be changed to hyphens throughout the documentation here. https://github.com/IMAP-Science-Operations-Center/ialirt-data-access/blob/78eb07b2879f5eaaa34d5cadfee86f7f9b0bba8c/pyproject.toml#L31